doc/man7/Ed25519.pod

```diff
@@ -15,16 +15,16 @@ draft-ietf-curdle-pkix-04.
 
 No additional parameters can be set during key generation one-shot signing or
 verification. In particular, because PureEdDSA is used, when signing or
-verifying a digest must B<NOT> be specified.
+verifying, a digest must B<NOT> be specified.
```

Comment:
I am not sure whether this comma is really necessary or not. Even if it is necessary, that sentence now contains far too many commas IMHO. You could fix this by changing the order as follows:

```
In particular, because PureEdDSA is used, a digest must B<NOT> be specified when signing or verifying.
```